### PR TITLE
[fix bug 1139716] Update Android product page to add Reading List

### PR DESCRIPTION
**Pull request by alexgibson**
Still waiting on some final assets and animation changes, so marking as WIP for now.

### CODE REVIEW

File: bedrock/firefox/templates/firefox/android/index.html [hunk 1]
```diff
@@ -22,9 +23,13 @@
 {% block site_header %}{% endblock %}
 
 {% block site_header_unwrapped %}
-  {% call fxfamilynav('dark') %}
+  {% call fxfamilynav() %}
     <a class="dl-button" id="nav-dl-button" rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}">
-      {{ _('Download on Google Play') }}
+      {% if l10n_has_tag('android_spring_updates') %}
+        {{ _('Get Firefox for android') }}
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/android/index.html]: Android (uppercase). We already have the string in [main.lang](http://transvision.mozfr.org/string/?entity=mozilla_org/main.lang:8156a2ee&repo=mozilla_org), so no need for tag.

File: bedrock/firefox/templates/firefox/android/index.html [hunk 2]
```diff
@@ -45,12 +50,20 @@ <h2 class="heading">
...
+        {% else %}
+          {{ _('Our most customizable Firefox for Android yet.') }}
+        {% endif %}
       </p>
 
       <div class="dl-button-wrapper">
         <a class="dl-button" rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}">
-          {{ _('Download on Google Play') }}
+          {% if l10n_has_tag('android_spring_updates') %}
+            {{ _('Get Firefox for android') }}
```
Comment (flodolo) [inline bedrock/firefox/templates/firefox/android/index.html]: Same here

File: bedrock/firefox/templates/firefox/android/index.html (hunk 1 above)
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/android/index.html]: Thanks @flodolo!

File: bedrock/firefox/templates/firefox/android/index.html [hunk 3]
```diff
@@ -345,7 +400,11 @@ <h2 class="heading">{{ _('Don’t leave home without being in Sync') }}</h2>
         <div id="download-content">
           <h3>{{ _('Choose Firefox') }}</h3>
           <a class="dl-button" rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}">
-            {{ _('Download on Google Play') }}
+            {% if l10n_has_tag('android_spring_updates') %}
+              {{ _('Get Firefox for android') }}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/android/index.html]: Same here

Comment (flodolo) [inline bedrock/firefox/templates/firefox/android/index.html]: Same here

**Comment by alexgibson** [comment]
Still waiting on confirmation of final strings & creative feedback, but this should at least be good for code review.

**Comment by alexgibson** [comment]
@jgmize  - this PR also adds a waffle switch `firefox-reading-list`

**Comment by alexgibson** [comment]
Changed the waffle switch on this PR from `firefox-reading-list` to `firefox-reader-view`

**Comment by alexgibson** [comment]
Ok this one is now ready for review. I've changed the waffle switch to use the more generic name `android-spring-updates`, since Firefox for Android is still going to ship with both Reader View & Reading List (it just doesn't use the Sync part).

**Comment by alexgibson** [comment]
Ok, this one should now be good for review :+1:

**Comment by schalkneethling** [comment]
I have checked this in Fx, Chrome and IE and it looks all good. Code looks good as well and, because the widget has been reviewed and tested in a separate PR, I am not concerned about it here.

So, r+ from me! Thanks @alexgibson :cake:

**Review by unforgiven211981 — COMMENTED** [review]
Michellle

**Pull request merged**